packages/react-devtools-shared/src/devtools/views/Settings/DebuggingSettings.js

```diff
@@ -35,7 +37,18 @@ export default function DebuggingSettings(_: {||}) {
           Append component stacks to console warnings and errors.
         </label>
       </div>
-
+      <div className={styles.Setting}>
+        <label>
+          <input
+            type="checkbox"
+            checked={enableDoubleLogging}
+            onChange={({currentTarget}) =>
+              setEnableDoubleLogging(currentTarget.checked)
```

Comment:
I think you'll want `setEnableDoubleLogging(!currentTarget.checked)` and `checked={!enableDoubleLogging}` since the meaning is the opposite. And then keep `false` as the default.

Alternatively, you can change all occurrences of the key to change the meaning as well.

Reply:
@gaearon i don't understand what you mean, don't you think it will change the course of things?

Reply:
Alright, still looking at it by the old name `Enable double logging`